javaagent-tooling/src/main/java/io/opentelemetry/javaagent/tooling/TracerInstaller.java

```diff
@@ -91,6 +91,7 @@ private static MetricExporterFactory findMetricExporterFactory(String exporterNa
       if (metricExporterFactory
           .getClass()
           .getSimpleName()
+          .replace("_", "")
```

Comment:
Hmm - feels a bit magic but I guess it's fine 😃

Reply:
yeah, maybe the factories could have a method that returns a name and do the comparison based on that.

Reply:
I like the idea of SpanExporterFactory/MetricExporterFactory having a method `Set<String> getNames()` (Set since e.g. OtlpSpanExporterFactory can be enabled via both `otlp` and `otlp_span`)

Reply:
I guess this is only applicable to our built-in exporters though, as opposed to someone building their own Exporter and using `EXPORTER_JAR`, so maybe not. Also, I forget, now that we are Java 8, we can always add interface methods after GA 😁

Reply:
Yeah I think my idea before was to have a map of wellknown names, and only use the classname for custom exporters